main/test/flix/Test.Exp.Jvm.InvokeMethod.VarArgs.flix

```diff
@@ -0,0 +1,17 @@
+mod Test.Exp.Jvm.InvokeMethod.VarArgs {
+
+    import java.nio.file.Path
+
+    @Test
+    def testInvokeVarArgs01(): Path \ IO =
+        Path.of("hello", (...{}: Vector[String]))
```

Comment:
IMO it's best if we require a literal vector here, to ensure no one else owns the vector and depends on its immutability.

But I guess that does not work for the empty case.

Reply:
Sadly the common case of empty varargs is a case we cannot easily support :(